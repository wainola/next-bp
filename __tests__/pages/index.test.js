const React = require('react')
const { shallow } = require('enzyme')

const { Home } = require('../../pages/index')

describe('<Home />', () => {
    it('it should render', () => {
        const component = shallow(<Home />)

        expect(component).toMatchSnapshot();
    })
})