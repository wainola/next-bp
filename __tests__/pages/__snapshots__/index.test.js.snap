// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> it should render 1`] = `
<div
  className="main-content"
>
  <_default />
  <h1
    className="main-title"
  >
    Hello World Jest NB!
  </h1>
</div>
`;
